package.json

```diff
@@ -32,16 +32,18 @@
   },
   "homepage": "http://gpii.net",
   "dependencies": {
+    "ajv": "^1.4.9",
+    "dedupe-infusion": "1.0.0",
     "first-discovery": "git+https://github.com/GPII/first-discovery.git",
```

Comment:
Either is fine - in general we should get into the habit of publishing more projects. We should adopt a general standard of all our npm artifacts being prefixed with gpii- , e.g. "gpii-first-discovery" in its package.json. if we decide to do this.